MAINTAINERS

```diff
@@ -81,7 +81,6 @@ S:	Orphan
 F:	core/arch/arm/plat-mediatek/
 
 NXP LS1021A, LS1043A-RDB, LS1046A-RDB, LS1012A-RDB, LS1012A-FRWY, LS1088A-RDB, LS2088A-RDB
-R:	Vinitha V Pillai <vinitha.pillai@nxp.com> [@nxa27423]
 R:	Pankaj Gupta <pankaj.gupta@nxp.com>
```

Comment:
While you're at it, would you mind adding your GitHub nickname here?
```
R:	Pankaj Gupta <pankaj.gupta@nxp.com> [@pangupta]
```

Reply:
Accepted.